test/integration/core/inspect_format.bats

```diff
@@ -23,7 +23,8 @@ load ${BASE_TEST_DIR}/helpers.bash
 }
 
 @test "$DRIVER: check .Driver output is not flawed" {
+  only_if_env DRIVER virtualbox
```

Comment:
Note the motivation for this change -- `SSHUser` is not consistent across all drivers. if our driver is `amazonec2` the default `SSHUser` will be `ubuntu`, for instance.